_posts/2021-12-27-pravega-connector-101.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Pravega Flink Connector 101"
+date: 2021-12-27 00:00:00
```

Comment:
```suggestion
date: 2021-01-02 00:00:00
```

Reply:
Perhaps schedule it to be published after the new year?

Reply:
Will that be `2022-01-02` ?

Reply:
Actually, I think we should try to schedule the publish date to be after the post about improving scheduler performance for large-scale jobs because that one is about <1.14 features.

Reply:
That's ok for me. Is there an estimated date for this?

Reply:
Let's try for January 10!